runtimes/peregrine/src/lib.rs

```diff
@@ -772,47 +780,132 @@ impl InstanceFilter<Call> for ProxyType {
 			ProxyType::Any => true,
 			ProxyType::NonTransfer => matches!(
 				c,
-				Call::System(..) |
-				Call::Scheduler(..) |
-				Call::Timestamp(..) |
-				Call::Indices(pallet_indices::Call::claim{..}) |
-				Call::Indices(pallet_indices::Call::free{..}) |
-				Call::Indices(pallet_indices::Call::freeze{..}) |
-				// Specifically omitting Indices `transfer`, `force_transfer`
-				// Specifically omitting the entire Balances pallet
-				Call::Authorship(..) |
-				Call::Session(..) |
-				Call::Democracy(..) |
-				Call::Council(..) |
-				Call::TechnicalCommittee(..) |
-				Call::TechnicalMembership(..) |
-				Call::Treasury(..) |
-				Call::Vesting(pallet_vesting::Call::vest{..}) |
-				Call::Vesting(pallet_vesting::Call::vest_other{..}) |
-				// Specifically omitting Vesting `vested_transfer`, and `force_vested_transfer`
-				Call::Utility(..) |
-				Call::ParachainStaking(..)
+				Call::Attestation(..)
+					| Call::Authorship(..)
+					// Excludes `Balances`
+					| Call::Council(..) | Call::Ctype(..)
+					| Call::Delegation(..)
+					| Call::Democracy(..)
+					| Call::Did(..)
+					| Call::DidLookup(..)
+					| Call::Indices(
+						// Excludes `force_transfer`, and `transfer`
+						pallet_indices::Call::claim { .. }
+							| pallet_indices::Call::free { .. }
+							| pallet_indices::Call::freeze { .. }
+					)
+					// Excludes `KiltLaunch`
+					| Call::ParachainStaking(..)
+					// Excludes `ParachainSystem`
+					| Call::Preimage(..)
+					| Call::Proxy(..)
+					| Call::Scheduler(..)
+					| Call::Session(..)
+					// Excludes `Sudo`
+					| Call::System(..)
+					| Call::TechnicalCommittee(..)
+					| Call::TechnicalMembership(..)
+					| Call::Timestamp(..)
+					| Call::Treasury(..)
+					| Call::Utility(..)
+					| Call::Vesting(
+						// Excludes `force_vested_transfer`, `merge_schedules`, and `vested_transfer`
+						pallet_vesting::Call::vest { .. }
+							| pallet_vesting::Call::vest_other { .. }
+					)
+					| Call::Web3Names(..),
+			),
+			ProxyType::NonDepositClaiming => matches!(
+				c,
+				Call::Attestation(
+						// Excludes `reclaim_deposit`
+						attestation::Call::add { .. }
+							| attestation::Call::remove { .. }
+							| attestation::Call::revoke { .. }
+					)
+					| Call::Authorship(..)
+					// Excludes `Balances`
```

Comment:
Do we really want to exclude Balances here?

Reply:
I would say we should not exclude balance transfers here. All calls should be allowed.